app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -88,7 +93,15 @@ async function* files_to_copy_provider() {
 
 const config: esbuild.BuildOptions = {
     bundle: true,
-    entryPoints: ['src/index.ts', 'src/wasm_imports.js'],
+    entryPoints: [
+        'src/index.ts',
+        'src/login.ts',
+        'src/registration.ts',
+        'src/confirmation.ts',
+        'src/forgot_password.ts',
+        'src/reset_password.ts',
+        'src/wasm_imports.js'
```

Comment:
make it one entry point, not in `content`, and dont use custom `esbuild config`.